web3.js

```diff
@@ -1,9 +1,9 @@
 const { getFilesFromPath } = require('files-from-path')
 const { Web3Storage } = require('web3.storage')
 
-async function addToWeb3 ({ endpoint, token, pathToAdd, name, wrapWithDirectory = false }) {
```

Comment:
Remove this unnecessary default.

Reply:
@ljmf00 I think this default is needed, since `web3.put` defaults `wrapWithDirectory` to `true`. So in this case we need actual `false`, not `undefined` :)

Reply:
Right, thanks, I was in doubt about this, hence my comment. I will change it.